Source/PropertyTools.Wpf/DataGrid/DataGrid.Strategy.cs

```diff
@@ -201,6 +203,31 @@ public override void GenerateColumnDefinitions()
                     return;
                 }
 
+                var view = CollectionViewSource.GetDefaultView(this.ItemsSource);

```

Comment:
I think it would be better to create a new "operator" class for this type of collections